src/jdk.jpackage/windows/classes/jdk/jpackage/internal/resources/WinResources_zh_CN.properties

```diff
@@ -57,6 +57,8 @@ error.unlock-resource=无法解锁：{0}
 error.read-wix-l10n-file=无法解析 {0} 文件
 error.extract-culture-from-wix-l10n-file=无法从 {0} 文件读取文化值
 error.short-path-conv-fail=Failed to get short version of "{0}" path
+error.missing-service-installer='service-installer.exe' service installer not found in the resource directory
+error.missing-service-installer.advice=Add 'service-installer.exe' service installer to the resource directory
```

Comment:
Same here.

Reply:
Should I revert changes in `WinResources_*.properties` files?

Reply:
Yes, please.

Reply:
Done